task11-next_js/pages/components/sipnosis.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary sipnosis component
+ */
+import React, { Component } from "react";
+import "./sipnosis";
+
+import mobileLogo from "../images/logo.svg";
+
+import SipnosisImage from "./sipnosisComponents/sipnosisImage";
+import CollapseCard from "./sipnosisComponents/collapseCard";
+
+class Sipnosis extends Component {
+  constructor(props) {
+    super(props);
+    this.openCard = this.openCard.bind(this);
+
+    this.state = {
+      cardIsOpen: false,
+    };
+  }
+
+  openCard() {
+    this.setState({ cardIsOpen: !this.state.cardIsOpen });
+  }
+
+  render() {
+    return (
+      <section className="sipnosis">
+        <SipnosisImage />
+        <img
+          className={`mobile__logo ${
+            this.state.cardIsOpen ? "mobile__logo--expanded" : ""
```

Comment:
You could use `this.state.cardIsOpen && "mobile__logo--expanded"` and avoid the empty string